arcgis-ios-sdk-samples/Layers/Group layers/LayersTableViewController.swift

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class LayersTableViewController: UITableViewController, GroupLayersCellDelegate, GroupLayersSectionViewDelegate {
+    var layers = [AGSLayer]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.register(GroupLayersSectionView.nib, forHeaderFooterViewReuseIdentifier: GroupLayersSectionView.reuseIdentifier)
```

Comment:
No. It is currently not possible to design a table header/footer view in a storyboard.

Reply:
Ooooh. That's collection views only? Didn't realize that (or had forgotten).

Reply:
I don't think it is only for collection views. It may be possible to design a header/footer view in a table/collection view with static cells, but it isn't possible to design a prototype header/footer view except in a xib..

Reply:
Hmm. I see. Yeah, this seems good. Sorry about that - I must not be remembering properly (I know I've done custom collection view header/footer stuff and don't remember calling register()).